zipline/finance/blotter.py

```diff
@@ -229,34 +229,37 @@ def get_transactions(self, bar_data):
             example, Zipline models the commission cost into the fill price
             of the transaction), then this is None.
         """
+
         closed_orders = []
         transactions = []
 
-        assets = self.asset_finder.retrieve_all(self.open_orders.keys())
-        asset_dict = {asset.sid: asset for asset in assets}
+        if len(self.open_orders) > 0:
```

Comment:
it's not.  I just noticed it as a small optimization and wanted to take care of it before I forgot.